lib/pf/Switch.pm

```diff
@@ -687,14 +581,14 @@ sub setVlan {
     # VLAN -1 handling
     # TODO at some point we should create a new blackhole / blacklist API
```

Comment:
Please go ahead and remove all the TODOs as you go along.
Most of them don't make sense anymore.